transport-native-epoll/src/test/java/io/netty/channel/epoll/EpollSocketConnectionAttemptTest.java

```diff
@@ -24,6 +24,6 @@
 public class EpollSocketConnectionAttemptTest extends SocketConnectionAttemptTest {
     @Override
     protected List<TestsuitePermutation.BootstrapFactory<Bootstrap>> newFactories() {
-        return EpollSocketTestPermutation.INSTANCE.clientSocket();
+        return EpollSocketTestPermutation.INSTANCE.clientSocket(false);
```

Comment:
Could we just add other methods for the "new behavior" and so not need to touch these files at all ?